src/components/RightSidebar/Participants/ParticipantsList/Participant/Participant.vue

```diff
@@ -143,6 +148,10 @@ export default {
 		isModerator() {
 			return this.participantTypeIsModerator(this.participantType)
 		},
+		showModeratorLabel() {
+			return this.isModerator
+				&& [CONVERSATION.TYPE.ONE_TO_ONE, CONVERSATION.TYPE.CHANGELOG].indexOf(this.conversation.type) === -1
```

Comment:
```suggestion
				&& [CONVERSATION.TYPE.ONE_TO_ONE, CONVERSATION.TYPE.CHANGELOG].includes(this.conversation.type)
```
May be clearer in this case